### DISCUSSION TITLE
Is there a roadmap about switching experimental to major release?

### DISCUSSION THREAD

**Discussion by NagaiKoki** [facebookexperimental/Recoil#1083] [Q&A]
I plan to adopt this lib as state management for production. However, I have concern about when this lib is going to release officially(I would avoid necessity of fixing bunch of code because of destructive changed after major release)

Please tell me the roadmap if there is or any information about release date roughly.

(I like this lib for state management because of simplicity, so I would like adopt if the elements of concern has been removed...!)

**Comment by knoefel** [comment]
+1 
I also really like to work with recoil, a major release would be really cool. Is there any news on this? Anyway, thanks for this great tool!!

**Comment by lalrg** [comment]
+1
Thanks for such an amazing tool! But yes, I also think it would be helpful to understand if there's going to be an official release in the future.
